jni/Android.mk

```diff
@@ -34,7 +34,7 @@ INIPARSER_SRCS := $(INIPARSER_ROOT)/src/iniparser.c \
 INIPARSER_INCLUDES := $(INIPARSER_ROOT)/src
 
 LOCAL_ARM_NEON      := true
-LOCAL_CFLAGS        += -pthread -fopenmp
+LOCAL_CFLAGS        += -pthread -fopenmp -fexceptions
```

Comment:
consider to add -frtti
https://developer.android.com/ndk/guides/cpp-support#rtti

Reply:
added!